ctapipe/conftest.py

```diff
@@ -218,7 +218,7 @@ def dl2_shower_geometry_file_type(dl2_tmp_path, prod5_gamma_simtel_path):
             f"--input={prod5_gamma_simtel_path}",
             f"--output={output}",
             "--write-images",
-            "--write-stereo-shower",
+            "--write-showers",
```

Comment:
Not sure I like the `showers` better than `stereo-shower`. Maybe go with `--write-dl2-geometry`?

Reply:
It's showers since it now will write both stereo and mono info that is in the DL2 container, which includes also energy and gammaness predictions if they exist (they don't currently of course since we haven't implemented those, but once they are the DataWriter will not need to change)

Reply:
What I don't like is this "mental mapping" between names and data levels. I think we should just call the options

```
--write-r1
--write-dl0
--write-dl1-images
--write-dl1-parameters
--write-dl2 (switches on geometry and future energy/gh separation, whatever is available)
```

Reply:
This PR changed the flags in a breaking way already...